src/SwerveModuleV2.cpp

```diff
@@ -33,37 +33,39 @@ SwerveModuleV2::SwerveModuleV2(uint32_t driveID, uint32_t steerID,
 	m_isMoving = false;
 	m_motionMagic = false;
 
-	m_driveMotor->SelectProfileSlot(0);
-	m_driveMotor->SetControlMode(CANTalon::kPercentVbus);
-	m_driveMotor->SetPID(SwerveModuleV2Constants::k_speedP,
-			SwerveModuleV2Constants::k_speedI,
-			SwerveModuleV2Constants::k_speedD);
-	m_driveMotor->SetF(0.1722);
-	m_driveMotor->SetIzone(200);
-	m_driveMotor->SetSensorDirection(true);
-	m_driveMotor->SetClosedLoopOutputDirection(true);
-
-	m_driveMotor->ConfigNominalOutputVoltage(0.0,0.0);
-	m_driveMotor->ConfigPeakOutputVoltage(12.0,-12.0);
+	m_driveMotor->SelectProfileSlot(0, 0);
+	m_driveMotor->Set(ControlMode::PercentOutput, 0);
+	m_driveMotor->Config_kP(0, SwerveModuleV2Constants::k_speedP, 0);
+	m_driveMotor->Config_kI(0, SwerveModuleV2Constants::k_speedI, 0);
+	m_driveMotor->Config_kD(0, SwerveModuleV2Constants::k_speedD, 0);
+	m_driveMotor->Config_kF(0, 0.1722, 0);
+	m_driveMotor->Config_IntegralZone(0, 200, 0);
+	m_driveMotor->SetSensorPhase(true);
+	m_driveMotor->SetInverted(true);
+
+	m_driveMotor->ConfigNominalOutputForward(0.0, 0.0);
+	m_driveMotor->ConfigNominalOutputReverse(0.0, 0.0);
+	m_driveMotor->ConfigPeakOutputForward(12.0, 0.0);
+	m_driveMotor->ConfigPeakOutputReverse(-12.0, 0.0);
 //	m_driveMotor->SetMotionMagicAcceleration(m_accel);
 //	m_driveMotor->SetMotionMagicCruiseVelocity(m_velocity);
 
-	m_driveMotor->ConfigNeutralMode(CANTalon::kNeutralMode_Brake);
+	m_driveMotor->SetNeutralMode(Brake);
 
-	m_steerMotor->SelectProfileSlot(0); //Profile 1 PIDf are P = 0.2 f = 1.1
-	m_steerMotor->ConfigNominalOutputVoltage(0,0);
-	m_steerMotor->SetControlMode(CANTalon::kPosition);
-	m_steerMotor->ConfigNeutralMode(CANTalon::kNeutralMode_Brake);
-	m_steerMotor->SetPID(SwerveModuleV2Constants::k_steerP,
-			SwerveModuleV2Constants::k_steerI,
-			SwerveModuleV2Constants::k_steerD);
-	m_steerMotor->SetSensorDirection(true);
-	m_steerMotor->SetClosedLoopOutputDirection(false);
-	m_steerMotor->SetPulseWidthPosition(m_steerMotor->GetPulseWidthPosition() & 0xFFF);
-	m_steerMotor->Enable();
-	m_steerMotor->SetAllowableClosedLoopErr(40);
-	m_steerMotor->SetStatusFrameRateMs(CANTalon::StatusFrameRateFeedback, 10);
-//	m_steerMotor->SetStatusFrameRateMs(CANTalon::StatusFrameRateGeneral, 10);
+	m_steerMotor->SelectProfileSlot(0, 0.0); //Profile 1 PIDf are P = 0.2 f = 1.1
+	m_steerMotor->ConfigNominalOutputForward(0,0);
+	m_steerMotor->ConfigNominalOutputReverse(0,0);
+	m_steerMotor->Set(ControlMode::Position, 0);
+	m_steerMotor->SetNeutralMode(Brake);
+	m_steerMotor->Config_kP(0, SwerveModuleV2Constants::k_steerP, 0);
+	m_steerMotor->Config_kI(0, SwerveModuleV2Constants::k_speedI, 0);
+	m_steerMotor->Config_kD(0, SwerveModuleV2Constants::k_speedD, 0);
```

Comment:
We need to pass in `k_steerI` and `k_steerD` in stead of `k_speedI` and `k_speedD`.